packages/btc/src/query/source.ts

```diff
@@ -35,14 +35,14 @@ export class DataSource {
     const txId = remove0x(hash);
     const tx = await this.service.getBtcTransaction(txId);
     if (!tx) {
-      return void 0;
+      return undefined;
```

Comment:
It is recommended to throw an exception directly here

Reply:
I looked at the reference of the function and it does look weird to throw afterward, I'll re-think about the signature of the function.